docker/templates/kaui/latest/kpm.yml

```diff
@@ -1,5 +1,6 @@
 ---
 kaui:
+  version: LATEST
   # Used for the sha1.yml
-  plugins_dir: /var/lib/kaui
+  plugins_dir: /var/lib/kaui/bundles
```

Comment:
Should we rely on `kaui_install_dir` ?

Reply:
This is a plain file, not a Jinja template. We could consider extending KPM to respect environment variables though (and use `$KAUI_INSTALL_DIR`).